go.mod

```diff
@@ -5,6 +5,7 @@ go 1.21.9
 // v0.8.0 was tagged long ago, and appared on pkg.go.dev.  We do not want any tagged version
 // to appear there.  The trick to accomplish this is to make a new version (in this case v0.9.0)
 // that retracts itself and the previous version.
+//fake
```

Comment:
```suggestion
```

Reply:
I just removed this commit which was just to trigger the tests and validate the upload was working